### PR TITLE
add .run_line_wrap helper function to wrap long CMD lines in dockerfiles

### PR DESCRIPTION
**Pull request by e-kotov**
Currently the `dockerize()` function produces rather unfriendly long lines of RUN commands. For example:
```{bash}
RUN apt-get update -qq && apt-get install -y libpcre3-dev zlib1g-dev pkg-config libcurl4-openssl-dev && apt-get install -y libcurl4-openssl-dev libicu-dev libssl-dev make zlib1g-dev
```

With the addition of the new internal (non-exported) helper function `.run_line_wrap()` this becomes a more conventional and human readable:
```{bash}
RUN apt-get update -qq \
	&& apt-get install -y libpcre3-dev zlib1g-dev pkg-config libcurl4-openssl-dev \
	&& apt-get install -y libcurl4-openssl-dev libicu-dev libssl-dev make zlib1g-dev
```

### CODE REVIEW

File: R/installation.R [hunk 1]
```diff
@@ -126,6 +126,11 @@
     invisible(readme)
 }
 
+.run_line_wrap <- function(command) {
```
Comment (chainsawriot) [inline R/installation.R]: In general, all functions in this package (helper functions incl.) are named with verb first. Or see #44 

`.wrap_run_line` is okay; but a better idea is `.generate_wrapped_line`: see above

```r
## Wrap long line by breaking line at &&
.generate_wrapped_line <- function(line) {
    gsub("&&", "\\\\\n\t&&", line)
}
```

Could you also provide tests? (Ideally with `writeLines`)

File: R/dockerfile.R [hunk 2]
```diff
@@ -76,7 +76,7 @@
 
 .write_dockerfile <- function(dockerfile_content, path) {
```
Comment (chainsawriot) [inline R/dockerfile.R:77]: Actually, I would argue that all directives (FROM, ENV, COPY, RUN, COPY, CMD) should be wrapped.

Therefore, it is possible to

```r
unlist(lapply(dockerfile_content, .run_line_wrap))
```

But again, we need tests to see how it works.

**Review by chainsawriot — CHANGES_REQUESTED** [review]

**Comment by chainsawriot** [comment]
@e-kotov Thanks a lot for the PR! I have provide the CR. Please let me know if you need help with the requested changes.

File: R/installation.R (hunk 1 above)
Comment (e-kotov) [reply R/installation.R]: Sounds great! Sorry, I did not pick up the convention form the code. I will do the test. Should I submit them via a separate PR?

Comment (chainsawriot) [reply R/installation.R]: @e-kotov no worries. Put the tests in the same PR if you can.

**Comment by e-kotov** [comment]
@chainsawriot I incorporated your review suggestions and written a test. I could not come up with the one that would use `writeLines()`. To me, it makes sense to check if all lines with `&&` start with `\t &&` and that there are no lines where there are two or more occurrences of `&&`.

**Review by chainsawriot — APPROVED** [review]

**Comment by chainsawriot** [comment]
@e-kotov Thanks a lot and I will add you as a CTB.

**Pull request merged**